package/MDAnalysis/analysis/nucleicacids.py

```diff
@@ -79,30 +100,47 @@ class NucPairDist(AnalysisBase):
     :class:`~MDAnalysis.core.groups.AtomGroup`.
 
     Parameters
-    __________
+    ----------
     selection1: List[AtomGroup]
-        list of :class:`~MDAnalysis.core.groups.AtomGroup` containing an atom
+        List of :class:`~MDAnalysis.core.groups.AtomGroup` containing an atom
         of each nucleic acid being analyzed.
-    selection1: List[AtomGroup]
-        list of :class:`~MDAnalysis.core.groups.AtomGroup` containing an atom
+    selection2: List[AtomGroup]
+        List of :class:`~MDAnalysis.core.groups.AtomGroup` containing an atom
         of each nucleic acid being analyzed.
     kwargs: dict
-        arguments for :class:`~MDAnalysis.analysis.base.AnalysisBase`
+        Arguments for :class:`~MDAnalysis.analysis.base.AnalysisBase`
 
     Attributes
-    __________
-        results: numpy.ndarray
-            first index is selection second index is time
-        results.times: numpy.ndarray
-            times used in analysis
+    ----------
+    results: numpy.ndarray
+        Array of pair distances. First index is selection, second index is time.
+
+        .. deprecated:: 2.4.0
+           Will be removed in MDAnalysis 2.5.0. Please use
+           :attr:`results.pair_distances` instead.
+
+    results.times: numpy.ndarray
+        Simulation times used in analysis
```

Comment:
I don't think we want `results.time` because we have the `times` attr in the class itself https://docs.mdanalysis.org/stable/documentation_pages/analysis/base.html#MDAnalysis.analysis.base.AnalysisBase.times

Reply:
Deprecating multiple things will get a little bit messy, one sec.

Reply:
Done, we should think of a cleaner way to deprecate things in the future, maybe we can just bake it into the base class?

Reply:
would be good

thanks for fixing this one